### ISSUE TITLE
Error occurred

### ISSUE THREAD

**Issue by immerzu** [RandomEngy/VidCoder#1225]
### What were you doing when VidCoder crashed?

Just wanted to recompress a film.

### Crash details

```text
VidCoder Beta 10.1 Portable
ReactiveUI.UnhandledErrorException: An object implementing IHandleObservableErrors (often a ReactiveCommand or ObservableAsPropertyHelper) has errored, thereby breaking its observable pipeline. To prevent this, ensure the pipeline does not error, or Subscribe to the ThrownExceptions property of the object in question to handle the erroneous case.
 ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IO.FileNotFoundException: Could not find file 'C:\Users\lolo\AppData\Roaming\VidCoder-Beta\Logs\2024-03-11 09.50.07 Scan list (800p_aac).txt'.
File name: 'C:\Users\lolo\AppData\Roaming\VidCoder-Beta\Logs\2024-03-11 09.50.07 Scan list (800p_aac).txt'
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at VidCoder.View.LogWindow.PopulateLogEntryChunks() in C:\git\VidCoder\VidCoder\View\LogWindow.xaml.cs:line 160
   at VidCoder.View.LogWindow.ConnectToLogger(IAppLogger newLogger) in C:\git\VidCoder\VidCoder\View\LogWindow.xaml.cs:line 117
   at VidCoder.View.LogWindow.<.ctor>b__16_3(LogViewModel selectedLog) in C:\git\VidCoder\VidCoder\View\LogWindow.xaml.cs:line 94
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/AnonymousSafeObserver.cs:line 54
   at System.Reactive.Linq.ObservableImpl.Switch`1._.InnerObserver.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Switch.cs:line 104
   at System.Reactive.Subjects.Subject`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/Subjects/Subject.cs:line 145
   at System.Reactive.Linq.ObservableImpl.SelectMany`2.EnumerableSelector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/SelectMany.cs:line 1375
   at System.Reactive.Linq.ObservableImpl.Buffer`2.Boundaries._.BufferClosingObserver.OnNext(TBufferClosing value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Buffer.cs:line 829
   at System.Reactive.Linq.ObservableImpl.Merge`1.Observables._.InnerObserver.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Merge.cs:line 240
   at System.Reactive.Subjects.Subject`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/Subjects/Subject.cs:line 145
   at ReactiveUI.IReactiveObjectExtensions.ExtensionState`1.NotifyObservable[T](TSender rxObj, T item, ISubject`1 subject) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\ReactiveObject\IReactiveObjectExtensions.cs:line 430
   at ReactiveUI.IReactiveObjectExtensions.ExtensionState`1.RaisePropertyChanged(String propertyName) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\ReactiveObject\IReactiveObjectExtensions.cs:line 385
   at ReactiveUI.IReactiveObjectExtensions.RaiseAndSetIfChanged[TObj,TRet](TObj reactiveObject, TRet& backingField, TRet newValue, String propertyName) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\ReactiveObject\IReactiveObjectExtensions.cs:line 121
   at VidCoder.Services.LogCoordinator.set_SelectedLog(LogViewModel value) in C:\git\VidCoder\VidCoder\Services\LogCoordinator.cs:line 53
   at VidCoder.View.LogWindow.<>c__DisplayClass16_0.<.ctor>b__5() in C:\git\VidCoder\VidCoder\View\LogWindow.xaml.cs:line 83
   at VidCoder.View.LogWindow.<.ctor>b__16_1(LogViewModel addedLogViewModel) in C:\git\VidCoder\VidCoder\View\LogWindow.xaml.cs:line 81
   at DynamicData.Kernel.EnumerableEx.ForEach[T](IEnumerable`1 source, Action`1 action) in D:\a\DynamicData\DynamicData\src\DynamicData\Kernel\EnumerableEx.cs:line 129
   at DynamicData.List.Internal.OnBeingAdded`1.RegisterForAddition(IChangeSet`1 changes) in D:\a\DynamicData\DynamicData\src\DynamicData\List\Internal\OnBeingAdded.cs:line 31
   at System.Reactive.Linq.ObservableImpl.Do`1.OnNext._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Do.cs:line 38
--- End of stack trace from previous location ---
   at System.Reactive.PlatformServices.ExceptionServicesImpl.Rethrow(Exception exception) in /_/Rx.NET/Source/src/System.Reactive/Internal/ExceptionServicesImpl.cs:line 19
   at System.Reactive.ExceptionHelpers.Throw(Exception exception) in /_/Rx.NET/Source/src/System.Reactive/Internal/ExceptionServices.cs:line 16
   at System.Reactive.Stubs.<>c.<.cctor>b__2_1(Exception ex) in /_/Rx.NET/Source/src/System.Reactive/Internal/Stubs.cs:line 16
   at System.Reactive.AnonymousSafeObserver`1.OnError(Exception error) in /_/Rx.NET/Source/src/System.Reactive/AnonymousSafeObserver.cs:line 62
   at System.Reactive.Sink`1.ForwardOnError(Exception error) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 60
   at System.Reactive.Linq.ObservableImpl.Do`1.OnNext._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Do.cs:line 42
   at System.Reactive.AutoDetachObserver`1.OnNextCore(T value) in /_/Rx.NET/Source/src/System.Reactive/Internal/AutoDetachObserver.cs:line 71
   at DynamicData.SourceList`1.<Connect>b__14_0(IObserver`1 observer) in D:\a\DynamicData\DynamicData\src\DynamicData\List\SourceList.cs:line 95
   at System.Reactive.Linq.QueryLanguage.CreateWithDisposableObservable`1.SubscribeCore(IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Creation.cs:line 35
   at System.Reactive.ObservableBase`1.Subscribe(IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/ObservableBase.cs:line 78
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Do`1.OnNext.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Do.cs:line 22
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 102
   at System.ObservableExtensions.Subscribe[T](IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 37
   at VidCoder.View.LogWindow..ctor() in C:\git\VidCoder\VidCoder\View\LogWindow.xaml.cs:line 77
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
   --- End of inner exception stack trace ---
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
   at System.Activator.CreateInstance(Type type)
   at VidCoder.Services.Windows.WindowManager.CreateWindow(Type viewModelType) in C:\git\VidCoder\VidCoder\Services\Windows\WindowManager.cs:line 767
   at VidCoder.Services.Windows.WindowManager.PrepareWindowForOpen(Object viewModel, Object ownerViewModel, Boolean userInitiated, Boolean isDialog) in C:\git\VidCoder\VidCoder\Services\Windows\WindowManager.cs:line 498
   at VidCoder.Services.Windows.WindowManager.OpenOrFocusWindow(Type viewModelType, Object ownerViewModel) in C:\git\VidCoder\VidCoder\Services\Windows\WindowManager.cs:line 294
   at VidCoder.Services.Windows.WindowManager.<>c__DisplayClass25_0.<CreateOpenCommand>b__0() in C:\git\VidCoder\VidCoder\Services\Windows\WindowManager.cs:line 374
   at ReactiveUI.ReactiveCommand.<>c__DisplayClass0_0.<Create>b__1(IObserver`1 observer) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\ReactiveCommand\ReactiveCommand.cs:line 109
   at System.Reactive.Linq.QueryLanguage.CreateWithDisposableObservable`1.SubscribeCore(IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Creation.cs:line 35
   at System.Reactive.ObservableBase`1.Subscribe(IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/ObservableBase.cs:line 58
   --- End of inner exception stack trace ---
   at ReactiveUI.RxApp.<>c__DisplayClass10_0.<.cctor>b__2() in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\RxApp.cs:line 106
   at System.Reactive.Concurrency.Scheduler.Invoke(Action action) in /_/Rx.NET/Source/src/System.Reactive/Concurrency/Scheduler.Simple.cs:line 253
   at System.Reactive.Concurrency.Scheduler.<>c.<Schedule>b__74_0(IScheduler _, Action a) in /_/Rx.NET/Source/src/System.Reactive/Concurrency/Scheduler.Simple.cs:line 37
   at System.Reactive.Concurrency.DispatcherScheduler.<>c__DisplayClass12_0`1.<Schedule>b__0() in /_/Rx.NET/Source/src/System.Reactive/Platforms/Desktop/Concurrency/DispatcherScheduler.cs:line 98
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by RandomEngy** [comment]
Thanks for the report. I'll have this fixed in the next Beta release.

This was caused by a log file being deleted.

**Comment by RandomEngy** [comment]
Fixed in 10.2 Beta
